docs/_static/f5-openstack-agent.gre.ini

```diff
@@ -81,13 +81,13 @@ periodic_interval = 10
 #  Device Setting
 ###############################################################################
 #
-# HA model
+# HA mode
 #
 # Device can be required to be:
 #
 # standalone - single device no HA
-# pair - active/standby two device HA(Not supported)
-# scalen - active device cluster(Not supported)
+# pair - active/standby two device HA
```

Comment:
I know talked to Jeff about this earlier, but is an Active/Active cluster referred as an HA Pair?